src/styles/blocks/card.scss

```
@@ -0,0 +1,98 @@
+.card {
+  background-color: #{$White};
+  border: 1px solid #f3f3f3;
+  border-radius: #{$Border-radius};
+  width: 200px;
+  padding: 2rem 1rem 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+
+  .card__img {
```

Please use [the sass ampersand](https://css-tricks.com/the-sass-ampersand/) wherever you can

```suggestion
  &__img {
```